File: python/7 kyu/83.py
# https://www.codewars.com/kata/65128732b5aff40032a3d8f0/train/python

# Passed

def neutralise(s1, s2):
    neutralised_string = str()
    for idx in range(len(s1)):
        if s1[idx] == s2[idx]:
            neutralised_string += s1[idx]
            continue
        neutralised_string += "0"

    return neutralised_string

output = neutralise("+++--+---", "++----++-")
print(output)

File: nodejs/8 kyu/83.js
// https://www.codewars.com/kata/5810085c533d69f4980001cf/train/javascript

// Passed

function calculator(a, b, sign) {
  operationsMap = {
    "+": (a, b) => a + b,
    "-": (a, b) => a - b,
    "*": (a, b) => a * b,
    "/": (a, b) => a / b,
  };
  if (
    (typeof a !== "number") |
    (typeof b !== "number") |
    !(sign in operationsMap)
  ) {
    return "unknown value";
  }

  const result = operationsMap[sign](a, b);
  return result;
}

const output = calculator(1, 2, "+");
console.log(output);